include/deal.II/base/symmetric_tensor.h

```diff
@@ -1667,13 +1694,42 @@ SymmetricTensor<rank,dim,Number>::operator [] (const TableIndices<rank> &indices
 
 
 
+
+namespace internal
+{
+  namespace SymmetricTensor
+  {
+    template <int dim, typename Number>
+    unsigned int
+    entry_to_indices (const dealii::SymmetricTensor<2,dim,Number> &,
```

Comment:
why does one need `const dealii::SymmetricTensor<4,dim,Number> &` if it is not used in either of the functions?